src/main/java/catan/services/impl/PlayServiceImpl.java

```diff
@@ -33,50 +33,22 @@
 
     private GameDao gameDao;
     private GameUtil gameUtil;
-    private EndTurnUtil endTurnUtil;
+    private PlayUtil playUtil;
 
     @Override
     public void buildRoad(UserBean user, String gameIdString, String edgeIdString) throws PlayException, GameException {
```

Comment:
reorganize calls of method and variable creation in the logically order (you don;t need edgeid at the begining, etc)